js/templates/modals/orderDetail/modFragment.html

```diff
@@ -9,3 +9,4 @@
   <% } %>
   <div><%= ob.name %><% print(modLink ? ` ${modLink}` : '') %></div>
 </div>
+<div class="js-verifiedMod"></div>
```

Comment:
![image](https://user-images.githubusercontent.com/794517/36861872-1869351c-1d3a-11e8-8175-b3015a97a377.png)

1.) That text doesn't really make sense in this context. You can't choose a verified moderator at this point. Perhaps the tooltip isn't needed here at all?
2.) Looks like some spacing is missing between that error icon and the text.
3.) Can we wrap the user name content and the verified mod content in a div and give it gutterVTn class so there is some vertical spacing between the name and the verified mod text?
4.) Instead of the emoji, can we use `ion-alert-circled clrTAlert`. It's a.) theme-able b.) consistent with what we use in the rest of the app and it's used a good amount. (I know the emoji might match the designs better, but everytime I bring up the consistency issue with @morebrownies, he opts for consistency over being 100% true to the designs).